velox/docs/develop/aggregate-functions.rst

```diff
@@ -92,12 +92,18 @@ Fixed-width part of the accumulator is stored in the row. Variable-width
 part (if exists) is allocated using :doc:`HashStringAllocator <arena>` and a pointer is
 stored in the fixed-width part.
 
-A row is a contiguous byte buffer. Given N aggregates, first N / 8 bytes
-store the null flags, one bit per aggregate, followed by fixed-width
-accumulators.
-
-.. image:: images/aggregation-layout.png
-  :width: 600
+A row is a contiguous byte buffer.  Data is stored in the following order:
```

Comment:
I see velox/docs/develop/images/aggregation-layout.png in the PR, but can't see what has changed.

Reply:
It was removed because it no longer reflects the current layout of the memory.

Reply:
Would it be possible to replace it with an accurate diagram. It helps to visualize the layout.

Reply:
The current layout is too complex to put on a diagram.  I think the new numbered list is more clear.

Reply:
The current layout for aggregation is the following:

- Null flags
- Free flag
- Keys
- Accumulators
- Size of variable-width accumulators

The previous diagram had null flags and accumulators. Why would adding free flag, keys and size of variable-width accumulators make it too complex?